### ISSUE TITLE
debs artifact "name" field looks to be being modified

### ISSUE THREAD

**Issue by awhitcroft** [canonical/test_observer#105]
The `name` field in the artifacts "table" entry for a debs entry looks to be being modified in an unexpected way. Note that the name has an `_` in it which is not a valid package name, and I presume was a `.` in the original:

  {
    "id": 12000,
    "name": "linux-generic-hwe-20_04",
    "version": "5.15.0.91.101~20.04.48",
    "track": null,
    "store": null,
    "series": "focal",
    "repo": "main",
    "stage": "updates",
    "status": "UNDECIDED",
    "assignee": null
  },

**Comment by jocave** [comment]
It's not being modified by test-observer - part of the Cert's team infrastructure does this and sends the artefact name with this change

I understand that this is not ideal for you, but would suggest that we track that against of test infra project rather than here.

**Comment by awhitcroft** [comment]
> It's not being modified by test-observer - part of the Cert's team infrastructure does this and sends the artefact name with this change
> 
> I understand that this is not ideal for you, but would suggest that we track that against of test infra project rather than here.

Anywhere is fine.  Can you elaborate on the modifications being done, if they are reversible then I can use that to recover the original.  (Right now I am assuming that only "." is replaced by "_" and therefore that it _is_ reversible, but I cannot be 100% sure obviously.

**Comment by jocave** [comment]
Yes exactly what you said "." are replaced with "_" as we had problems with a component that relies on JSONPaths, but we could not seem to escape the "." chars in package names.

**Comment by omar-selo** [comment]
This should no longer be the case anymore. We've modified cert team's infrastructure to send the name as is without modification to Test Observer.
